client/components/empty-content/feature-unavailable/index.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+import { domainManagementEdit } from 'my-sites/upgrades/paths';
+
+const FeatureUnavailable = ( { selectedSite, translate } ) => {
+	return (
+		<EmptyContent
+			className={ 'feature-unavailable' }
+			title={ translate( 'Add a site to start using this feature.' ) }
+			line={ translate( 'Your domain is only set up with a temporary page. Start a site now to unlock everything WordPress.com can offer.' ) }
+			action={ translate( 'Create Site' ) }
+			actionURL={ `/start/site-selected/?siteSlug=${ encodeURIComponent( selectedSite.slug ) }&siteId=${ encodeURIComponent( selectedSite.ID ) }` }
```

Comment:
I disagree. `site.ID` will not be anything but integer for the foreseeable future. Wrapping this in encode sends the message to developer that it might be a string *today*.